.circleci/config.yml

```diff
@@ -953,6 +953,15 @@ jobs:
                 when: always
                 command: |
                   ./Tests/scripts/install_content_and_test_integrations.sh "$INSTANCE_ROLE"
+            - run:
```

Comment:
This will currently run at every build - is this what we want?
Should it run in every nightly, or every master build instead?

Reply:
This will only run on nightly 
                 `if [ -n "${NIGHTLY}" ] ;
                    then
                        ./Tests/scripts/validate_premium_packs.sh "$INSTANCE_ROLE"
                  fi`

I tried using when: <<<< pipeline.parameters.nightly >>>> but it didn't work. Maybe you know a way?

Reply:
The `pipeline.parameters.nightly` will work only if the workflow was triggered manually, otherwise we don't set it